CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# CHANGELOG for `process_settings`
+
+Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.6.1] (Unreleased)
+
+### Added
+
+- Don't check for file_path or logger config if `instance =` has already been called;
```

Comment:
I think this would be a `Modified` wouldn't it?

Reply:
Good point! Fixed.